__tests__/utils.js

```diff
@@ -0,0 +1,17 @@
+import {makePromise} from '../src/utils'
+
+describe('makePromise', () => {
```

Comment:
I'd like you to also write tests for those resolve and reject functions to make sure they're attached properly and do what they're supposed to do. One way of doing this with jest would be to write it like this:

```javascript
describe('resolve function', () => {
  it('resolves a promise', (done) => {
    const promise = makePromise().then(() => done())
    promise.resolve()
  })
})
```